script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -635,89 +748,102 @@ She hasn't even seen them yet, but the fact that an uninvolved outsider arrived
 -- Page 472, Offset 19448.
 -- 「だがまあ、ちょいとやり過ぎだな。
-"But eh, that was a bit much.
+"But man, you really went ham.
 }
 [sha:3e105f8a8b962c011d530e7616cda72b3ec42ff6]{
 -- Page 472, Offset 19449.
 -- 死徒どもに懺悔の機会なんぞを与えやがったのは見なかった事にする。代行者の殺し方はそれぞれだからな。
-I'll overlook the fact that you gave the Dead Apostles a chance at redemption. Each Executor has their own way of killing after all.
+I'll pretend I didn't see you offer those Dead Apostles a shot at redemption.
+ Each Executor's got their own particular flavor of murder after all.
 }
 [sha:8140564cea29305a68e6eb223b64c7de1cd3bd04]{
 -- Page 472, Offset 19450.
 -- 問題は死体の数だ。中にはまだ成りきれてない犠牲者もいたように見えたがねぇ」
-The problem lies with how many you killed. Looks like some of the victims weren't even completely transformed yet."
+What I can't ignore is how %{i}many%{/i} you killed.
+ Looked like some of 'em weren't even totally finished turnin' yet either."
 }
 [sha:85da08ccc3b60c3ece466eebec2a8c33b42f0617]{
 -- Page 473, Offset 19451.
 -- 「いいえ、アレは吸血鬼より始末が悪いものです。
-"No, they are even harder to deal with than vampires.
+"I stand by my actions. People like that are even more pernicious than vampires.
 }
 [sha:d84e637a4655d76b2a34bfa9613a01e9008b46e2]{
 -- Page 473, Offset 19452.
 -- あの少女は人間であるにもかかわらず、自分から奴等に協力して犠牲者を増やしていましたから。当然の末路です」
-That girl, despite being a human, chose to collaborate with them, increasing the number of victims. It was a foregone conclusion."
+Despite being human, she chose to work with them and was responsible for increasing the number of victims.
+ Her fate was already sealed."
 }
 [sha:8024e5426c21aef29fe7ad033eaac7838dfedd95]{
 -- Page 474, Offset 19453.
 -- 「ほう。チキンなテメェにしちゃあ冷徹な判断だ。吸血鬼に関わった人間は皆殺しってワケか。そいつは頼もしい」
-"Ooh? That's a cold-blooded decision for a coward like you. To kill every human involved with vampires, huh? That's one reliable method."
+"Oh yeah?
+ That's a pretty cold-blooded decision for a chicken-shit like you.
+ You're just gonna kill any human that gets involved with vampires, huh?
+ I mean hey, it's a reliable strategy."
 }
 [sha:98baae8744a5c52c88c062f9b9a12fbe2c98790e]{
 -- Page 474, Offset 19454.
 -- 「……何のご用でしょうか、司祭代行。
-"...If I may ask, why are you here, Acting Presbyter?
+"If I may ask... why are you here, Acting Presbyter?
 }
 [sha:63f566c56f8bf7e73c1565100670ca85b5a39ccd]{
 -- Page 474, Offset 19455.
 -- 定期連絡はしていますし、わたしの単独行動は許可されているのではないのですか？」
-I have maintained regular contact. Besides, am I not allowed to conduct operations on my own?"
+I have maintained regular contact, and I'm allowed to act independently, aren't I?"
 }
 [sha:80170603f29b66235da16b3d024429894db64f5e]{
 -- Page 475, Offset 19456.
 -- 「ああ、秘蹟会と埋葬機関は別管轄だ。テメェがシエルの相棒である以上、オレたちは対等<だ|・><っ|・><た|・>」
-"Ahh, the Sacrament Assembly and the Burial Agency are two separate jurisdictions. When you were Ciel's partner, we were on equal footing."
+"Yeah, the Sacrament Assembly and the Burial Agency are two separate jurisdictions.
+ As Ciel's partner, you and I %{i}were%{/i} on equal footing."
```

Comment:
There are probably multiple Sacrament Assemblies, so this is most likely a plural. Hard to tell for sure though.
Maybe with Mario's informal register you can kinda avoid using 'the' altogether, though?
>Yeah, Sacrament Assembly's jurisdiction is separate from the Burial Agency.

Reply:
Well, plural it is then.